src/config.js

```diff
@@ -111,12 +124,26 @@ const setConfig = async cfg => {
 
     config.jwsVerificationKeys = {};
 
+    // read files on startup.
     fs.readdirSync(cfg.JWS_VERIFICATION_KEYS_DIRECTORY)
         .filter(f => f.endsWith('.pem'))
         .map(f => {
             config.jwsVerificationKeys[path.basename(f, '.pem')] = fs.readFileSync(path.join(cfg.JWS_VERIFICATION_KEYS_DIRECTORY, f));
         });
 
+    // continuously monitor folder for changes in files.
+    fsWatcher = fs.watch(cfg.JWS_VERIFICATION_KEYS_DIRECTORY, (eventType, filename) => {
+        // On most platforms, 'rename' is emitted whenever a filename appears or disappears in the directory.
+        // From: https://nodejs.org/docs/latest/api/fs.html#fs_fs_watch_filename_options_listener
+        if (eventType === FS_EVENT_TYPES.RENAME) {
+            if (config.jwsVerificationKeys[path.basename(filename, '.pem')] == null) {
```

Comment:
Unsure whether `===` beats `==` for comparison with `null` but perhaps worth using anyway.

Reply:
It will not work, as if you see two lines below that I use `delete` to completely remove that value, so strictly checking for `null` would return `false` as the value is actually  `undefined`.

This is the only case that I don't use triple `=`, as it is kind of safe since it is truthy only for `null` and `undefined` values. That is why eslint doesn't complain as well, while in other cases it would.

Reply:
Great, thanks